test/lambda-createLambdaExecutionEnvironment-errorHandling.test.js

```diff
@@ -3,16 +3,25 @@ const sinon = require('sinon');
 const Docker = require('dockerode');
 const path = require('path');
 const lambda = require('../src/lambda');
+const assert = require('assert');
 
-test('Cleanups up the network and container on failure after start', async function (test) {
+let sandbox;
+test.beforeEach(() => {
+  sandbox = sinon.createSandbox();
```

Comment:
Is there a reason you chose to use a module scoped variable instead of a `test.context` variable? It seems like a `test.context` variable would be the more traditional thing to use.

Reply:
oh, thanks, I didn't realize that. I will update.